2024/COMMUNICATION.md

```diff
@@ -0,0 +1,27 @@
+# Communication
+
+## Discourse announcement
+
+Dear Nix Community,
+
+It is with excitement that we launch 2024's Nix Survey!
+
+https://survey.nixos.org/2024
+
+We are conducting this survey to improve our understanding of the community and what everyone cares about.
+We hope to use your responses to support Nix Teams in best serving the community.
+
+As always, the survey is **anonymous**, and aggregated statistics and major findings will be published on Discourse.
```

Comment:
As always, the survey is **anonymous**. Aggregated statistics and major findings will be published on Discourse.

Reply:
```suggestion
As always, the survey is **anonymous**
Aggregated statistics and major findings will be published on Discourse.
```